src/corelibs/U2View/src/ov_msa/General/GeneralTabOptionsPanelWidget.ui

```diff
@@ -6,7 +6,7 @@
    <rect>
```

Comment:
Changes in this file are mostly due to the QTCreator update. There is 1 real change here from me: an extra button "to Amino"